.github/workflows/ci.yml

```diff
@@ -9,13 +9,16 @@ jobs:
         # Adding additional destinations, schemes, etc. here will make GitHub run the steps
         # on all combinations of them (e.g., if there are 2 destinations and 2 schemes, the
         # test suite will get run 4 times).
-        destination: ["platform=iOS Simulator,OS=17.2,name=iPhone 15"]
+        destination: ["platform=iOS Simulator,OS=17.4,name=iPhone 15"]
         scheme: ["Thumbprint"]
-        xcode: ["/Applications/Xcode_15.2.app"]
+        xcode: ["/Applications/Xcode_15.3.app"]
     steps:
       - name: 📥 Checkout
         uses: actions/checkout@v2
 
+      - name: Select Xcode
+        run: sudo xcode-select -s "/Applications/Xcode_15.3.app"
```

Comment:
I don't think you can do this ? These are githubs machines, not ours.

Reply:
it's the work around proposed in https://stackoverflow.com/a/73896628/695599 and it fixed the lint issue we are seeing

Reply:
it's very strange to me that you can run sudo at all here 🤔

Reply:
yeah we need sudo and it's proposed by the actions :)
<img width="1060" alt="Screenshot 2024-04-25 at 12 12 07 PM" src="https://github.com/thumbtack/thumbprint-ios/assets/113396903/992d7880-b882-45e9-b621-3b292b7e4220">

Reply:
it's not proposed by the actions, but by xcode-select. It doesn't know it's run from github.

Reply:
I guess they say it's fine 🤷‍♂️ 

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#administrative-privileges

Reply:
if we don't do that we get lint failure. maybe we can force merge and on the next pr we won't have this issue as we did already upgraded to xcode 15.3? 
<img width="1085" alt="Screenshot 2024-04-25 at 12 17 26 PM" src="https://github.com/thumbtack/thumbprint-ios/assets/113396903/a083fbbe-219a-4dcd-89a2-7765e24fff84">

Reply:
The vms aren't dedicated though, so i think you'd run into it again. I think eventually you'd be able to drop it when 5.10 is the default they have etc.